src/main/resources/assets/quilt-loading-screen/lang/en_us.json

```diff
@@ -0,0 +1,4 @@
+{
+  "quilt-loading-screen.midnightconfig.title": "\u00a7lQuilt Loading Screen",
+  "quilt-loading-screen.midnightconfig.prideQuiltsEnabled": "Enable Pride Quilts even when it's not June!"
+}
```

Comment:
newline expected at end of file

Reply:
Doesn't really matter, json files work just fine without one